src/app/component/about-page/about-page.component.html

```diff
@@ -0,0 +1,161 @@
+<div class="container">
```

Comment:
I guess it's not related to this ticket, please revert it

Reply:
fixed